
package com.sforce.soap.partner;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.2.6
 * Thu Feb 11 15:37:56 EST 2010
 * Generated source version: 2.2.6
 * 
 */

@WebFault(name = "UnexpectedErrorFault", targetNamespace = "urn:fault.partner.soap.sforce.com")
public class UnexpectedErrorFault extends Exception {
    public static final long serialVersionUID = 20100211153756L;
    
    private com.sforce.soap.partner.fault.UnexpectedErrorFault unexpectedErrorFault;

    public UnexpectedErrorFault() {
        super();
    }
    
    public UnexpectedErrorFault(String message) {
        super(message);
    }
    
    public UnexpectedErrorFault(String message, Throwable cause) {
        super(message, cause);
    }

    public UnexpectedErrorFault(String message, com.sforce.soap.partner.fault.UnexpectedErrorFault unexpectedErrorFault) {
        super(message);
        this.unexpectedErrorFault = unexpectedErrorFault;
    }

    public UnexpectedErrorFault(String message, com.sforce.soap.partner.fault.UnexpectedErrorFault unexpectedErrorFault, Throwable cause) {
        super(message, cause);
        this.unexpectedErrorFault = unexpectedErrorFault;
    }

    public com.sforce.soap.partner.fault.UnexpectedErrorFault getFaultInfo() {
        return this.unexpectedErrorFault;
    }
}
